### DISCUSSION TITLE
Is there Specific Reason for targeting .net8.0-windows instead of just .net8.0?

### DISCUSSION THREAD

**Discussion by BadWolf97** [MahApps/MahApps.Metro.IconPacks#352] [Q&A]
Hello,

is there a Specific Reason for targeting .net8.0-windows instead of just .net8.0? This makes it nearly impossible to use the Project in Uno Platform. In my (to be honest, very simple) test it should build without the "-windows" as well.

I'm happy for insights, and thank you for this project!
Nico

**Comment by BadWolf97** [comment]
Nevermind, it just looked like that due to a glitch with Visual Studio. But is there another way to target Uno or MAUI?

**Reply by Insire** [reply]
It says 
![grafik](https://github.com/user-attachments/assets/90eab735-4601-4f91-a557-1dee68343d15)
in the readme. Both are windows only.
